keras/src/layers/attention/multi_head_attention.py

```diff
@@ -392,7 +393,13 @@ def _masked_softmax(self, attention_scores, attention_mask=None):
         return self._softmax(attention_scores, mask=attention_mask)
 
     def _compute_attention(
-        self, query, key, value, attention_mask=None, training=None
```

Comment:
Add a MHA layer constructor argument (set to `None` as default) for `flash_attention`.

None -> use global default
True, False -> overrides global default